### ISSUE TITLE
Unexpected Message Error in email alert notifications

### ISSUE THREAD

**Issue by tmanh17** [doitintl/bigquery-grafana#329]
## Bug Report
Get error in email alert notifications:
tsdb.HandleRequest() response error &{json: cannot unmarshal object into Go struct field queryModel.group of type string json: cannot unmarshal object into Go struct field queryModel.group of type string A <nil> [] [] 0xc000eaf6b0}
## Expected Behavior
Get the abnormal metric name and its value  
## Actual Behavior
Get the Error message ```tsdb.HandleRequest() response error &{json: cannot unmarshal object into Go struct field queryModel.group of type string json: cannot unmarshal object into Go struct field queryModel.group of type string A <nil> [] [] 0xc000eaf6b0}``` instead of the metric name and its value
## Steps to Reproduce the Problem

1.create a new dashboard
2.create a panel with BigQuery plugin
3.create a BigQuery query
4.create an email channel and alert to that channel
5.check the email alert notification 

![image](https://user-images.githubusercontent.com/29015535/116498075-15aa9a00-a8d3-11eb-8b76-3a707ba67b03.png)

## Specifications

- Version: 2.0.1
- Platform: Debian 8.5
- Grafana Version: Grafana v7.4.2

**Comment by tmanh17** [comment]
Some logs
```
t=2021-04-29T17:06:16+0700 lvl=eror msg="Alert Rule Result Error" logger=alerting.evalContext ruleId=4 name=Alert error="tsdb.HandleRequest() error could not find executor for data source type: doitintl-bigquery-datasource" changing state to=alerting

t=2021-04-29T17:06:30+0700 lvl=eror msg="Alert Rule Result Error" logger=alerting.evalContext ruleId=5 name="Panel Title alert" error="tsdb.HandleRequest() error could not find executor for data source type: doitintl-bigquery-datasource" changing state to=alerting
```

**Comment by ofir5300** [comment]
Duplication of #301 which is solved by #322 
Should work properly on the next plugin release v2.0.2
